gptme/util.py

```diff
@@ -148,9 +168,16 @@ def ask_execute(question="Execute code?", default=True) -> bool:  # pragma: no c
     termios.tcflush(sys.stdin, termios.TCIFLUSH)  # flush stdin
 
     choicestr = f"[{'Y' if default else 'y'}/{'n' if default else 'N'}]"
+    copystr = r"\[c] to copy" if copiable else ""
     answer = console.input(
-        f"[bold bright_yellow on red] {question} {choicestr} [/] ",
+        f"[bold bright_yellow on red] {question} {choicestr}{copystr} [/] ",
     )
+    if copiable and "c" in answer.lower():
```

Comment:
It should still be:

```suggestion
    if copiable and "c" == answer.strip().lower():
```

Reply:
See previous comment. Although I don't think it's that common of a use case